.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+   build:
+     docker:
+       - image: ubuntu:xenial
+     steps:
+       - checkout
+       - run:
+           name: Install system requirements
+           command: |
+             apt-get update
+             apt-get install -y software-properties-common python-software-properties
+       - run:
+           name: Install python3.6 and pip
```

Comment:
Using tox makes sens, I suggest to have one job for each python version.

specific env can be selected using `tox -e py27`